oaknational/Oak-Web-Application
# Fix/cur 262/use mv curriculum units 4

## Description
- On units tab of curriculum visualiser, point to new version of API endpoint: mv_curriculum_units_4
- Eliminate duplicate and incorrect units in sequence
- Eliminate duplicates in unit's and unit options' lesson lists
- Filter unit options by examboard when available
- Fix ordering of year 10 and 11 Science child subjects

## How to test

1. Go to https://deploy-preview-1981--oak-web-application.netlify.thenational.academy/teachers/curriculum/english-secondary-aqa/units
2. You should see unique units with no duplicates
3. Click a unit with several unit options
4. You should see unique unit options with no duplicates
5. Go to {owa_deployment_url}/teachers/curriculum/science-secondary-ocr/units
6. You should see unique units with no duplicates
7. For years 10 and 11, you should the child subjects in the correct order: `Combined Science` > `Biology` > `Physics` > `Chemistry`

## Screenshots

How it used to look (delete if n/a):
<img width="993" alt="Screenshot 2023-10-05 at 17 16 01" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/7fcc802d-ac5d-46c5-9ab6-96c4575b6528">
<img width="993" alt="Screenshot 2023-10-05 at 17 15 28" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/1a3f4f0c-2f41-42bb-839e-3f7a4d420d1d">

How it should now look:
<img width="993" alt="Screenshot 2023-10-05 at 17 15 55" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/5e33f733-3a24-415d-b987-e0b8b1a8bd09">
<img width="993" alt="Screenshot 2023-10-05 at 17 15 13" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/dd77c20d-92f5-4829-9fad-09664ddb30cd">

## Checklist
